src/main/java/com/sample/project/demo/entity/UserDetails.java

```diff
@@ -0,0 +1,15 @@
+package com.sample.project.demo.entity;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.Id;
+import java.time.ZonedDateTime;
+
+public class UserDetails {
+    @Id
+    @GeneratedValue
+    private int userDetailsId;
+    private String userName;
```

Comment:
change userName to username.
 because in the liquibase definition <column name="username" type="varchar(255)" />
userName would've been accepted if <column name="user_name" type="varchar(255)" /> was defined